devenv/docker/blocks/postgres/docker-compose.yaml

```diff
@@ -7,10 +7,18 @@
     ports:
       - "5432:5432"
     command: postgres -c log_connections=on -c logging_collector=on -c log_destination=stderr -c log_directory=/var/log/postgresql
+    healthcheck:
```

Comment:
While you're at it, can you rename the service from `postgrestest` to `postgres`. Have been thinking about that for long, but haven't made the change. Currently the postgres_tests block also defines a service named `postgrestest` so that's why I would like to change this.